divvy/twisted_client.py

```diff
@@ -45,6 +46,8 @@ def connect(self):
         if self.connection:
             self.disconnect()
         self.connection = self._makeConnection()
+        if self.debug_mode:
+            self.log.debug('DivvyClient: {msg}', msg='Connecting to server')
```

Comment:
don't really need a substitution here

Reply:
Want to check #connects and #drops and test persistence.